SWLOR.Game.Server/Service/QuestService/NPCGroupType.cs

```diff
@@ -66,18 +66,18 @@ public enum NPCGroupType
         Hutlar_QionSlugs = 29,
         [NPCGroup("Qion Tigers")]
         Hutlar_QionTigers = 30,
-        [NPCGroup("Tuk'ata")]
-        Korriban_Tukata = 31,
-        [NPCGroup("Hssiss")]
-        Korriban_Hssiss = 32,
+        [NPCGroup("Pelko Bug Swarm")]
+        Korriban_PelkoSwarm = 31,
```

Comment:
This change will break in-progress quests. Again, we track data on entities by enumeration name. Simplest approach is to change these back to their original names. Most complex is to perform a data migration.